docs/digger/admin/installing-jenkins-plugins.asciidoc

```diff
@@ -0,0 +1,21 @@
+== Installing Jenkins plugins
+Digger will by default install a number of Jenkins plugins required for its proper operation. If more Jenkins plugins are required, they can be specified in Ansible playbook(???) or using the Jenkins user interface.
+
+NOTE: Plugins for Jenkins can be found at link::https://plugins.jenkins.io. (PLEASE VERIFY LINK SYNTAX)
```

Comment:
Oh yeah, this should just be `NOTE: Plugins for Jenkins can be found at https://plugins.jenkins.io`.